node/coinstacks/solana/api/src/controller.ts

```diff
@@ -126,4 +127,30 @@ export class Solana implements BaseAPI {
       throw handleError(err)
     }
   }
+
+  /**
+   * Get the current recommended gas fees to use in a transaction
```

Comment:
```suggestion
   * Get the estimated compute unit cost of a transaction
```

Reply:
Changed in [d50be64](https://github.com/shapeshift/unchained/pull/1051/commits/d50be6415f5b5822023b295cfe71938ed521ffb5)